src/RequestExecutor/HttpRequestExecutor.ts

```diff
@@ -106,6 +126,39 @@ export class HttpRequestExecutor implements RequestExecutor {
     }).on('request', (req: OutgoingMessage) => this.setHeaders(req, options));
   }
 
+  private async verifyCertificate(filename: string): Promise<void> {
+    const AVAILABLE_CERTIFICATES = ['.ca', '.pfx'];
+    const extension = extname(filename);
+    if (!AVAILABLE_CERTIFICATES.includes(extension)) {
+      throw new Error(`File of type ${extension} does not support.`);
+    }
+    try {
+      await fileExists(filename);
+
+      return;
+    } catch (e) {
+      throw new Error(`Certificate ${e.path} not found.`);
```

Comment:
As I told you before, you have to show the warning. Don't raise errors during request execution.